frontend/docs/admin-guide/install.md

```diff
@@ -1061,6 +1061,7 @@ Use the MLSpace Config Wizard by running `npm run config` and select "Advanced C
 | `PROJECTS_TABLE_NAME` |  Dynamo DB table to hold project related metadata | `mlspace-projects` |
 | `PROJECT_USERS_TABLE_NAME` |  Dynamo DB table to hold project membership related metadata. Including permissions and project/user specific IAM role data. | `mlspace-project-users` |
 | `USERS_TABLE_NAME` |  Dynamo DB table to hold user related metadata | `mlspace-users` |
+| APP_CONFIGURATION_TABLE_NAME |  Dynamo DB table to hold dynamic configuration settings. These are settings than can be modified after the app has been deployed. | `mlspace-app-configuration` |
```

Comment:
We'll also want [a section for the table here](https://awslabs.github.io/mlspace/admin-guide/getting-started.html#dynamodb) eventually but I we don't need to do that in this PR since this is just foundational.